src/Traits/Resource/HasWebhooks.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
Traits are for repeatable code.
If there are more Kubernetes resources that can have webhooks, just resolve this review, as this trait can be useful in the future.

Reply:
I created this trait because it will be used for the mutation webhook PR that will come in the next few days 🚀🔥👀